larray/core/array.py

```diff
@@ -8027,63 +8027,69 @@ def transform(v):
         return self.apply(transform, dtype=dtype)
 
 
+class LArray(Array):
+    def __init__(self, *args, **kwargs):
+        warnings.warn("LArray has been renamed as Array.", FutureWarning, stacklevel=2)
+        Array.__init__(self, *args, **kwargs)
```

Comment:
can't we use `LArray = renamed_to(Array, 'LArray')` instead?

Reply:
``rename_to`` return a function and not a class. In addition, the printed warning message would have been `LArray() is deprecated. Use Array() instead`, i.e. with ( ) after class names.

Reply:
Calling a function and a class is essentially the same thing in Python (they are both callables) and I don't see the problem with that warning message.